### DISCUSSION TITLE
Disable bluetooth by default on Ledger Nano X

### DISCUSSION THREAD

**Discussion by phantomcrypt** [LedgerHQ/ledger-live-desktop#3864] [Ideas]
I believe bluetooth should be disabled by default as security precaution.

In my case, I do not even use it, so I'd like to always keep it disabled. However when it comes the time to upgrade the device, all settings are lost, which means bluetooth is back to being enabled.

**Comment by Imonda** [comment]
I agree with you. Bluetooth is not safe especially with "safe" wallet.
